sunscreen_runtime/debugger-frontend/src/App.tsx

```diff
@@ -200,25 +276,69 @@ const App = () => {
           { source: 1, target: 2, directed: true, arrowhead: 'normal' }
         ]
       }
-      setGraph(lineNumber !== 1 ? graph : exGraph)
-    }, [setLine, setGraph]
+      setGraph(lineNumber !== 1 ? graph : dataToGraph(await fetch(`/sessions/${session}`).then(d => d.json())))
+    }, [setLine, setGraph, session]
```

Comment:
I don't think `setLine` or `setGraph` do anything here as dependencies. They are just functions, so they don't change. Also it looks like currently this will either reload the entire graph or just give a hardcoded graph, so maybe you should remove this for now?